app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,60 @@
+class ArticlesController < ApplicationController
+  before_action :find_article, except: %i[index new create]
+
+  http_basic_authenticate_with name: 'user', password: 'secret', except: %i[index show]
+
+  def index
+    @articles = Article.all
+  end
+
+  def show
+    @article 
+  end
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+    begin
+      @article.save!
+    rescue StandardError => e
+      render :new
+    else
+      redirect_to(article_path(@article))
```

Comment:
Why did you include the `else` here?

Reply:
I consider this from [How to Rescue Exceptions](https://stackify.com/rescue-exceptions-ruby/)
![image](https://user-images.githubusercontent.com/24459154/114455298-353a8680-9ba1-11eb-904d-a3b5e3a72fe7.png)

Should I use the `redirect_to` by default without use `else` ?

```
def create
  ....
  redirect_to article_path(@article)
end
```

Reply:
See https://github.com/SanGeeky/rails-training/pull/1/files#r612082199